### PR TITLE
Adding Meredith Palmer to team page

### LINKED ISSUE #116: Add Meredith to SS Team Page
Can someone add Meredith Palmer please? 

Meredith Palmer is a graduate student studying large mammal communities in Tanzania and South Africa. After spending a few years conducting fieldwork around the world, she began her Ph.D. at the University of Minnesota in 2013. She is interested in predator-prey interactions, particularly what ungulates do to avoid becoming lunch. When not plodding through data analyses at the University of Minnesota, she can be found in Serengeti digging Land Rovers out of the mud and continuing Ali's work with the camera traps. 
p1130667

### PR DESCRIPTION
**Pull request by mrniaboc**
Closes #116

### CODE REVIEW

**Comment by chrislintott** [comment]
@chrissnyder Can we merge this?

**Comment by chrissnyder** [comment]
Probably, but I get a bunch of errors when trying to run it locally and can't look at it or deploy. Maybe @alexbfree when you have a moment?

**Comment by mrniaboc** [comment]
@alexbfree Can you check if this break stuff, and if not, please merge?

File: app/translations/en_us.coffee [hunk 1]
```diff
@@ -1132,6 +1132,13 @@ module.exports =
             He’s interested in how social and cultural context influence how people learn about science, and conducts much of his research in Thailand.
           '''
 
+        palmer:
+          name: 'Fred Finlay'
```
Comment (alexbfree) [inline app/translations/en_us.coffee]: @aliburchard surely this should say Meredith Palmer not Fred Finlay?

**Comment by alexbfree** [comment]
@mrniaboc sorry i missed your question in December on this - didn't see a notification about it.

**Comment by alexbfree** [comment]
This has now been handled by the following PRs #140, #142, #143. So it can be closed. 

@mrniaboc I recommend you delete this branch, and recreate your master (or your whole fork) from the main branch since your master seems to be out of sync somehow, lots of geordi diffs showing up here.

**Pull request closed without merging**